membership-attribute-service/app/controllers/AccountController.scala

```diff
@@ -274,6 +277,7 @@ class AccountController(commonActions: CommonActions, override val controllerCom
   }
 
   private def updateContributionAmount(subscriptionNameOption: Option[memsub.Subscription.Name]) = BackendFromCookieAction.async { implicit request =>
```

Comment:
this function is not deprecated - just the endpoint of the same name is, in favour of `/updateAmountForSpecificContribution` see https://github.com/guardian/members-data-api/blob/09ab6bafaa0b77eb8106b78568ee7911347f961a/membership-attribute-service/conf/routes#L12 and https://github.com/guardian/members-data-api/blob/09ab6bafaa0b77eb8106b78568ee7911347f961a/membership-attribute-service/app/controllers/AccountController.scala#L305

Reply:
think you might want to do an if statement a bit like the `lookup` one you've done

Reply:
Ooof, good point. See: https://github.com/guardian/members-data-api/pull/392/commits/b1bfde67af5a835f0645283bebe8b2baaa7582cd